justfile

```diff
@@ -247,6 +247,32 @@ assets-test: assets-install
     npm run test:coverage
 
 
+# Build a lightweight setup for local development purposes with test fixture data.
+build-dbs-for-local-development nuclear="":
+    #!/usr/bin/env bash
+    set -euo pipefail
+
+    # WARNING: Passing 'nuclear' to this just recipe will delete
+    # `db.sqlite3` if it exists in the project root.
+    if [ -n "{{ nuclear }}" ]; then
+        echo "Nuclear option enabled: deleting db.sqlite3"
+        rm -f db.sqlite3
+        # Set up or update the local dev environment:
+        # - Recreates core DB and applies migrations
+        just dev-setup
```

Comment:
What was your thinking about referencing this here rather than at the start of the recipe? E.g.

`build-dbs-for-local-development nuclear="": dev-setup`